extractor/text.go

```diff
@@ -105,13 +106,18 @@ func (e *Extractor) extractPageText(contents string, resources *model.PdfPageRes
 				// Begin a text object, initializing the text matrix, Tm, and the text line matrix,
 				// Tlm, to the identity matrix. Text objects shall not be nested; a second BT shall
 				// not appear before an ET.
-				if to != nil {
+				if inTextObj {
 					common.Log.Debug("BT called while in a text object")
 				}
+				inTextObj = true
 				to = newTextObject(e, resources, gs, &state, &fontStack)
 			case "ET": // End Text
+				if !inTextObj {
+					common.Log.Debug("ET called outside of a text object")
```

Comment:
Add a comment regarding behavior when ET is called outside of a text object.
Would there be any marks available in this case? etc?  Good to state how we handle this case.
Possibly also when nesting BT inside another BT block (not allowed by standard).